pkg/proxy/accesslog/record.go

```diff
@@ -195,3 +198,22 @@ type LogRecordHTTP struct {
 	// Header is the HTTP header in use
 	Header http.Header
 }
+
+// LogRecordKafka contains the kafka specific portion of a log record
+
+type LogRecordKafka struct {
+	// Version of the kafka api used
+	apiVersion int
+
+	// Kafka ApiKey
+	// Reference: https://kafka.apache.org/protocol#protocol_api_keys
+	apiKey int16
+
+	// correlation_id is a user-supplied integer value that will be passed
+	// back with the response
+	correlation_id int32
+
```

Comment:
any specific reason why client id is not included?

Reply:
Per discussion with  Romain, since  client ID is a very dynamic value, it might not be quite useful.